tests/integration/targets/aci_bfd_multihop_interface_profile/tasks/main.yml

```diff
@@ -0,0 +1,174 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Anvitha Jain (@anvjain)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: debug
+
+- name: Query system information
+  cisco.aci.aci_system:
+    <<: *aci_info
+    id: 1
+    state: query
+  register: version
+
+- name: Execute tasks only for non-cloud sites
+  when: version.current.0.topSystem.attributes.version is version('5.2', '>=')  # This condition will execute only when APIC version >= 5.2
+  block:  # block specifies execution of tasks within, based on conditions
+
+  # CLEAN ENVIRONMENT
+  - name: Remove the ansible_tenant
+    cisco.aci.aci_tenant:
+      <<: *aci_info
+      tenant: ansible_tenant
+      state: absent
+
+  # Add a tenant
+  - name: Add a new tenant
+    cisco.aci.aci_tenant:
+      <<: *aci_info
+      tenant: ansible_tenant
+      description: Ansible tenant
+      state: present
+
+  # Add VRF
+  - name: Add a new VRF
+    cisco.aci.aci_vrf:
+      <<: *aci_info
+      tenant: ansible_tenant
+      name: ansible_vrf
+      description: Ansible VRF
+      state: present
+
+  # Add L3Out
+  - name: Add a new L3 Outside
```

Comment:
cloud sites fail: "text": "fvTenant:  cannot contain: object of class l3extOut with RN (Dn0), DN is (Rn) Dn0=uni/tn-ansible_tenant/out-ansible_l3out, Rn=out-ansible_l3out,